### PR TITLE
bitwise 0.33 (New Formula)

### PR DESCRIPTION
**Pull request by kavu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bitwise.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Bitwise < Formula
+  desc "Terminal based bit manipulator in ncurses"
+  homepage "https://github.com/mellowcandle/bitwise"
+  url "https://github.com/mellowcandle/bitwise/releases/download/v0.32/bitwise-v0.32.tar.gz"
+  sha256 "9dbc6646e685083cd8b876c41e828e1a4a1a6ddcb1b9765b0ef76c7fbb97f6c6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "readline"
```
Comment (fxcoudert) [inline Formula/bitwise.rb]: Can this use macOS libeditline?

File: Formula/bitwise.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bitwise", "0x29A"
```
Comment (fxcoudert) [inline Formula/bitwise.rb]: Check the output with `assert_match` please

Comment (kavu) [reply Formula/bitwise.rb]: Done! Updated PR.

File: Formula/bitwise.rb (hunk 1 above)
Comment (kavu) [reply Formula/bitwise.rb]: Unfortunately, no. We will get many errors like `error: use of undeclared identifier 'rl_display_prompt';`. But I may miss something here.

**Comment by kavu** [comment]
Changed to 0.33 version.

**Comment by kavu** [comment]
@fxcoudert Sorry for bothering. Any chance on merge?

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
Thanks @kavu

**Pull request closed without merging**